crypto/fipsmodule/ec/p256-nistz_test.cc

```diff
@@ -41,7 +41,15 @@
 
 TEST(P256_NistzTest, SelectW5) {
   // Fill a table with some garbage input.
-  alignas(64) P256_POINT table[16];
+  // A bug on aarch not fixed in gcc 7 and 9 prevents the alignas usage.
```

Comment:
May be we can add "in cpp files where the argument to alignas is "seemingly" larger than the size of the array being aligned (faulty check)".
What do you think?

Reply:
Updated the comment.